src/app/game/game.component.ts

```diff
@@ -1,23 +1,38 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, HostListener, OnInit, OnDestroy } from '@angular/core';
+import { Subscription } from 'rxjs';
 
-import { Answer } from '../models/answer';
-import { AnswerService } from '../services/answer.service';
+import { GamePlayService } from '../services/game-play.service';
 
 @Component({
   selector: 'app-game',
   templateUrl: './game.component.html',
   styleUrls: ['./game.component.scss']
 })
-export class GameComponent implements OnInit {
-  protected answer: Answer = { id: 0, word: '' };
-  constructor(private answerService: AnswerService) { }
+export class GameComponent implements OnInit, OnDestroy {
+  constructor(private gameplayService: GamePlayService) { }
+  protected currentGuess = '';
+  protected acceptedGuesses: string[] = []
+  protected currentGuess$?: Subscription;
```

Comment:
In lieu of making a `Subscription` here, I'd remove this and remove line 13. Then you can just reference `this.gameplayService.currentGuess$` inside the keydown listener.

I think adding in a `Subscription` just adds more complexity and is generally a code-smell to me. 

The same can be done for `acceptedGuesses$` and `acceptedGuesses` as well